src/data/markdown/docs/03 cloud/03 Manage/06 Audit Trail.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Audit Trail'
+excerpt: 'Monitor actions performed by your organization'
+slug: '/cloud/manage/audit-trail'
+---
+
+The *Audit Trail* can provide valuable insights into the activity taking place within your organization.
+
+> ⭐️ Audit Trail is available on annual Pro and Enterprise plans.
+
+To view the Audit Trail, go to  **Manage > Audit Trail** section.
+
+## What is an Audit trail
```

Comment:
```suggestion
## What is an Audit trail?
```

Reply:
5 weeks ago, I would've agreed. But in that time, I've started to be wary of putting questions in sub-headings, because people scan pages looking at the left side of the column. So, placing "What is" in the column makes the content a little more difficult to scan.

Not my idea, by the way. I stole it from here: https://content-guide.18f.gov/our-approach/structure-the-content/
And it made sense to me, since people [read online in F-shaped patterns](https://www.nngroup.com/articles/f-shaped-pattern-reading-web-content/).

Does it make sense?

Reply:
After your changes, it makes more sense.